partner_communication_switzerland/tests/test_bug_fixes.py

```diff
@@ -49,9 +67,16 @@ def test_co_1272(self, get_pdf, update_hold):
             dossier communication for the sponsor.
         """
         update_hold.return_value = True
-        f_path = 'addons/partner_communication_switzerland/static/src/test.pdf'
-        with file_open(f_path) as pdf_file:
-            get_pdf.return_value = pdf_file.read()
+
+        if self.is_travis:
+            travis_path = 'addons/' + self.base_path
+            with file_open(travis_path, 'rb') as test:
+                get_pdf.return_value = test.read()
+        else:
+            cwd = os.getcwd()
+            f_path = cwd + '/compassion-switzerland/' + self.base_path
+            with open(f_path, 'rb') as fopen:
+                get_pdf.return_value = fopen.read().decode('latin-1')
```

Comment:
This should not be treated differently. The method `file_open` in odoo tools should normally correctly locate the addons folder and be able to import the file. Please have a look at the odoo method and how to correctly use it, as it should work without doing such hacks. Plus, your code doesn't work on my computer, as I have a different directory structure. It's not a good practice to hardcode the paths like this. It should always be relative to the module.

Reply:
Ok, I had the impression that something had changed, but I will rollback to the old way and make it work !